lead-a-session.md

```diff
@@ -1,16 +1,18 @@
 ---
 title: Lead a session
 layout: page
-has-nav: lead-a-session
+# has-nav: lead-a-session
 ---
 
-<em>Submissions are now closed for SPA Software in Practice 2018.</em>
+<!-- <em>Submissions are now closed for SPA Software in Practice 2018.</em> -->
 
 <h1>Lead a session</h1>
-<p>{{ site.conference.name }} is about practitioners from any aspect of software development - technology, people and process - sharing their thoughts and knowledge, and learning together.</p>
-<p>All the sessions are interactive - everyone can be an active participant and learn by doing.</p>
+<p>The CfP for {{ site.conference.name }} will open around the 15th of November 2018. Please email <a href="mailto:conference@spaconference.org?subject=NotifySPA2019">conference@spaconference.org</a> with if <i>NotifySPA2019</i> in the subject if you want to be notified.
```

Comment:
@gasproni @simplytalking Instead of getting people to email, you could suggest that they sign up for a SPA account and make sure the box for receving communication is checked? On https://spaconference.org/scripts/editmyprofile.php. We usually send out an email to everyone who has checked that box when the CFP is open anyway, so that saves extra work.

That would also correspond with what it says under 'session proposals' on the profile page https://spaconference.org/scripts/myprofile.php